ovs/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # CMake build file for Open vSwitch
 #
-# Copyright 2022-2023 Intel Corporation
+# Copyright 2022-2024 Intel Corporation
```

Comment:
Update copyright date in other files of this PR (if they havent already been updated to 2024)

Reply:
I think this is the last hold-out.